app/src/main/java/org/mozilla/fenix/tabtray/TabTrayViewHolder.kt

```diff
@@ -72,7 +73,10 @@ class TabTrayViewHolder(
         if (tab.thumbnail != null) {
             thumbnailView.setImageBitmap(tab.thumbnail)
         } else {
-            imageLoader.loadIntoView(thumbnailView, ImageRequest(tab.id))
+            // Make sure we have the view's dimensions so we can load the image at the correct size
+            thumbnailView.doOnNextLayout {
+                imageLoader.loadIntoView(thumbnailView, tab.id)
+            }
```

Comment:
Alternatively we could hardcode the image size to 100dp instead of using the view dimensions through the extension function. In my testing loading with the extension function requests a slightly smaller image (253px vs 275px on Pixel 3 emulator)